src/build.ts

```diff
@@ -21,20 +22,21 @@ function getModuleNameForPath(path: string): string {
 /**
  * get the command to call for the package
  */
-function getModuleCommandForPath(path: string): string {
+function getModuleCommandForPath(path: string): string | void {
   const packageJson = JSON.parse(
     fs.readFileSync(`${cwd}/${path}/package.json`).toString()
   );
 
-  // a command override is found in the package
-  if (packageJson['watch-module'] && packageJson['watch-module']['command']) {
-    return packageJson['watch-module']['command'];
-  }
+  const moduleConfig = getModuleConfigEntry(path);
+  const command = moduleConfig.command;
 
   // no command override found
-  const yarnOrNpm = hasYarn(path) ? 'yarn' : 'npm';
+  if (command && packageJson['scripts'] && packageJson['scripts'][command]) {
```

Comment:
according to the readme, the command will be `yarn run build:prod` here, so the "real" command will be `yarn run yarn run build:prod`.

I think if a command is defined as configuration, we should not assume that it's a yarn / npm command. You might want to just do shell

Reply:
Ok, even better